{{cookiecutter.project_name}}/MANIFEST.in

```diff
@@ -1,8 +1,10 @@
-include AUTHORS.md
 include CONTRIBUTING.md
+include README.md
+{% if cookiecutter.open_source_project|lower == "y" %}
```

Comment:
You are missing here `CONTRIBUTING.md`

Reply:
Contributing is useful for internal projects as well because it contains instructions how to setup development environment.

Reply:
BitBucket for example recognizes this file and shows a special button in a PR "Contribution guidelines"